package.json

```diff
@@ -35,7 +34,7 @@
     "test:debug": "node --debug-brk --inspect ./node_modules/.bin/jest -i",
     "test:karma": "karma start --single-run",
     "lint": "eslint .",
-    "build": "webpack --mode=production",
+    "build": "NODE_OPTIONS=--openssl-legacy-provider webpack --mode=production",
```

Comment:
Without this flag, the build errors:

```
node:internal/crypto/hash:71
  this[kHandle] = new _Hash(algorithm, xofLen);
                  ^

Error: error:0308010C:digital envelope routines::unsupported
    at new Hash (node:internal/crypto/hash:71:19)
    at Object.createHash (node:crypto:133:10)
    at module.exports (/Users/joeybaldwin/scm/sd/equivalency/node_modules/webpack/lib/util/createHash.js:135:53)
    at NormalModule._initBuildHash (/Users/joeybaldwin/scm/sd/equivalency/node_modules/webpack/lib/NormalModule.js:417:16)
    at handleParseError (/Users/joeybaldwin/scm/sd/equivalency/node_modules/webpack/lib/NormalModule.js:471:10)
    at /Users/joeybaldwin/scm/sd/equivalency/node_modules/webpack/lib/NormalModule.js:503:5
    at /Users/joeybaldwin/scm/sd/equivalency/node_modules/webpack/lib/NormalModule.js:358:12
    at /Users/joeybaldwin/scm/sd/equivalency/node_modules/loader-runner/lib/LoaderRunner.js:373:3
    at iterateNormalLoaders (/Users/joeybaldwin/scm/sd/equivalency/node_modules/loader-runner/lib/LoaderRunner.js:214:10)
    at iterateNormalLoaders (/Users/joeybaldwin/scm/sd/equivalency/node_modules/loader-runner/lib/LoaderRunner.js:221:10)
    at /Users/joeybaldwin/scm/sd/equivalency/node_modules/loader-runner/lib/LoaderRunner.js:236:3
    at context.callback (/Users/joeybaldwin/scm/sd/equivalency/node_modules/loader-runner/lib/LoaderRunner.js:111:13)
    at /Users/joeybaldwin/scm/sd/equivalency/node_modules/babel-loader/lib/index.js:55:71
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
  library: 'digital envelope routines',
  reason: 'unsupported',
  code: 'ERR_OSSL_EVP_UNSUPPORTED'
}

```

Reply:
Interesting that webpack fails. Why would it be using crypto? 

Do you think this could be fixed by upgrading webpack? Doesn't have to be in this PR, just curious if that's your take.

Reply:
Based on this issue, it would still be an issue for webpack 5 but may not be an issue for webpack 6 in the future.
https://github.com/webpack/webpack/issues/14532#issuecomment-947525539

If we were to upgrade to webpack 5, we would be able to manually configure the hashing algorithm so it is not md4:
https://webpack.js.org/configuration/output/#outputhashfunction

But we'd have to monkey patch it for webpack 4.